twoliter/embedded/Makefile.toml

```diff
@@ -186,6 +186,15 @@ BUILDSYS_OVF_TEMPLATE = "${BUILDSYS_ROOT_DIR}/variants/${BUILDSYS_VARIANT}/templ
 # The default name of uploaded OVAs; override by setting VMWARE_VM_NAME
 VMWARE_VM_NAME_DEFAULT = "${BUILDSYS_NAME}-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-v${BUILDSYS_VERSION_IMAGE}-${BUILDSYS_VERSION_BUILD}"
 
+# Depends on ${BUILDSYS_ARCH} and ${BUILDSYS_VARIANT}.
+FETCH_VARIANT_LOCAL_ROOT_JSON = { script = ['echo "${FETCH_VARIANT_LOCAL_ROOT_JSON:-""}"'] }
+FETCH_VARIANT_ROOT_JSON = { script = ['echo "${FETCH_VARIANT_ROOT_JSON:-https://cache.bottlerocket.aws/root.json}"'] }
+FETCH_VARIANT_ROOT_JSON_SHA512 = { script = ['echo "${FETCH_VARIANT_ROOT_JSON_SHA512:-a3c58bc73999264f6f28f3ed9bfcb325a5be943a782852c7d53e803881968e0a4698bd54c2f125493f4669610a9da83a1787eb58a8303b2ee488fa2a3f7d802f}"'] }
+
+FETCH_VARIANT_UNIFIED = { script = ['echo "${FETCH_VARIANT_UNIFIED:-false}"'] }
+FETCH_VARIANT_METADATA_URL = { script = ['echo "${FETCH_VARIANT_METADATA_URL:-https://updates.bottlerocket.aws/2020-07-07/"${BUILDSYS_VARIANT}"/"${BUILDSYS_ARCH}"/}"'] }
+FETCH_VARIANT_TARGETS_URL = { script = ['echo "${FETCH_VARIANT_TARGETS_URL:-https://updates.bottlerocket.aws/targets}"'] }
```

Comment:
No. This needs to integrate with `Infra.toml` and take a named repo. We have `PUBLISH_REPO` for the target of a new repo build, so you can have `FETCH_REPO` for the source of an import.